examples/dask/special-topics-rolling-averages.ipynb

```diff
@@ -0,0 +1,197 @@
+    "\n",
+    "* Sort by index within AND across partitions\n",
+    "* Know when to compute (convert to pandas DF) or persist (process computations on cluster)\n",
+    "* Run calculations, with attention to our need to cross partitions correctly.\n",
+    "\n",
+    "This example will walk you through these specific points, and demonstrate how it's done. We'll use New York City taxi trip data, and get the 30-day rolling average of base fare prices, for our example."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "622fb142-cee7-4659-a9b5-d9cbde6bf912",
+   "metadata": {},
+   "source": [
+    "## Single Node"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c831ea89-1530-404f-8acc-94d3c28a603b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "096acff8-a6a7-4df0-a971-0c18a483a783",
+   "metadata": {},
+   "source": [
+    "## Cluster"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "462c9dea-fe10-4fa4-8120-15ee89caf7ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_saturn import SaturnCluster\n",
+    "from dask.distributed import Client\n",
+    "\n",
+    "cluster = SaturnCluster(\n",
+    "    scheduler_size=\"medium\",\n",
+    "    worker_size=\"xlarge\",\n",
+    "    n_workers=3,\n",
+    "    nthreads=4,\n",
+    ")\n",
+    "client = Client(cluster)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "280f72b9-9602-460d-9aa0-9caf3999773b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import s3fs\n",
+    "\n",
+    "s3 = s3fs.S3FileSystem(anon=True)\n",
+    "files_2019 = \"s3://nyc-tlc/trip data/yellow_tripdata_2019-*.csv\"\n",
+    "s3.glob(files_2019)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ddf40e08-c3ad-4679-ae1b-3887f4fd60ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask.dataframe as dd\n",
+    "\n",
+    "taxi = dd.read_csv(\n",
+    "    files_2019,\n",
+    "    parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"],\n",
+    "    storage_options={\"anon\": True},\n",
+    "    assume_missing=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3baefab-6733-4b41-abc0-63a89d645fe8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "taxi = taxi.set_index(\"tpep_pickup_datetime\")\n",
+    "from dask.distributed import wait\n",
+    "\n",
+    "taxi = taxi[\"2019-01-01\":\"2020-01-01\"]\n",
+    "taxi = taxi.persist()\n",
+    "_ = wait(taxi)"
+   ]
+  },
```

Comment:
There should be text describing what this chunk is doing.

also the import statement should be at the top.